src/modules/user/dto/user.dto.ts

```diff
@@ -12,4 +12,16 @@ export default class UserDto {
 
   @ApiProperty({ example: '' })
   profile_image: string;
+
+  @ApiProperty({ example: '' })
+  available_time: string;
+
+  @ApiProperty({ example: '' })
+  position: string;
+
+  @ApiProperty({ example: '' })
+  hourly_rate: number;
+
+  @ApiProperty({ example: '' })
```

Comment:
frontend will use it as documentation, it should be described as good as possible

Reply:
fixed